zhaquirks/xiaomi/aqara/light_acn.py

```diff
@@ -200,3 +226,27 @@ class LumiLightAcn014(XiaomiCustomDevice):
             },
         }
     }
+
+
+(
+    QuirkBuilder(AQARA, "lumi.light.acn032")
+    .friendly_name(manufacturer="Aqara", model="Ceiling Light T1M")
+    .replaces(AqaraLightT1M)
+    .replaces(AqaraLightT1M, endpoint_id=2)
+    .enum(
+        AqaraLightT1M.AttributeDefs.power_on_state.name,
+        LumiPowerOnStateMode,
+        AqaraLightT1M.cluster_id,
+        translation_key="power_on_state",
+        fallback_name="Power on state",
+    )
+    .enum(
+        AqaraLightT1M.AttributeDefs.power_on_state.name,
+        LumiPowerOnStateMode,
+        AqaraLightT1M.cluster_id,
+        translation_key="power_on_state",
+        fallback_name="Power on state",
+        endpoint_id=2,
```

Comment:
Ok, I'll just move up the `endpoint_id` argument, as we try to have `translation_key` and `fallback_name` as the last arguments to be somewhat consistent between quirks.
```suggestion
    .enum(
        AqaraLightT1M.AttributeDefs.power_on_state.name,
        LumiPowerOnStateMode,
        AqaraLightT1M.cluster_id,
        endpoint_id=2,
        translation_key="power_on_state",
        fallback_name="Power on state",
```